src/docs/telemetry.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Telemetry
+description: Stencil Telemetry usage information
+url: /telemetry
+contributors:
+  - splitinfinities
+---
+
+# Stencil CLI telemetry
+
+Stencil collects anonymous telemetry data about usage of our command line interface. Participation in this program is optional, and [you may opt-out](#how-do-i-opt-out) if you'd prefer not to share any information.
```

Comment:
This is more a question for Josh - should we say "as of version 2.7.0...." here?

Reply:
Added